patroni/dcs/consul.py

```diff
@@ -135,6 +135,10 @@ def connect(self, *args, **kwargs):
             kwargs['token'] = self._token
         return HTTPClient(**kwargs)
 
+    def set_token(self, token):
+        if token:
```

Comment:
We just have to set the value, without performing any checks.

Reply:
Understood & fixed